kedro-telemetry/tests/test_masking.py

```diff
@@ -29,6 +29,7 @@
     "ipython",
     "jupyter",
     "lint",
+    "micropkg",
```

Comment:
Mm have you merged latest `main` in your branch? I updated this in the last PR. 🤔

Reply:
yes it's picked up now!